GCRCatalogs/dc2_truth_match.py

```diff
@@ -1,13 +1,18 @@
 """
 Reader for truth-match catalogs persisted as parquet files and partitioned in tracts.
 """
+import os
+
 import numpy as np
 import astropy.units as u
 
 from .dc2_dm_catalog import DC2DMTractCatalog
 
 __all__ = ["DC2TruthMatchCatalog"]
 
+FILE_DIR = os.path.dirname(os.path.abspath(__file__))
+META_PATH = os.path.join(FILE_DIR, 'catalog_configs/_dc2_truth_match_meta.yaml')
```

Comment:
Just an observation:  there seems to be no way to deal with schema versioning, but in this way truth_match is no different from other meta descriptions. Maybe this possibility is handled somewhere else?

Reply:
Because of the design philosophy of "maximum flexibility", it is entirely left to each reader to decide to how to implement the quantity info. 

The quantity info should be embedded could be embedded in the data files themselves, in which case they are automatically versioned (like the case of cosmoDC2 --- it reads in quantity info directly from the hdf5 file instead of a meta file).  

For object catalog and alike, we opted to use a meta yaml file checked in the github repo. If the same reader wants to access different meta files, we can define multiple `META_PATH` and the reader can pick on based on, say, an option in the catalog config.

Reply:
Ok.  I guess what I would have expected for those readers which use an external metafile would be something like your last remark: there could be a recognized option in the the config file, call it "meta_path" (ideally the same string for any such reader).   If it was there, the reader would use its value; otherwise it would use a default.   Currently it's not an issue since there is no need for an alternate.

Reply:
Agreed! Implemented now in #522.